exercises/practice/automated-readability-index/.docs/instructions.md

```diff
@@ -0,0 +1,115 @@
+# Instructions
+
+As we grow up, we can understand more and more complex texts. But how to assess 
+the level of complexity of a given text, and how to teach the computer to do so? 
+In this exercise, you will write a program determining how difficult a text is 
+and for what age it is best suited.
+
+For a formula for calculating the complexity, we will use 
+[Automated readability index (ARI)][wiki] 
+developed in 1967 year. The ARI assesses the U.S. grade level required to read 
+a piece of text. In some ways, it is similar to other formulas. Its difference 
+is rather than counting syllables, it counts characters. The more characters, 
+the harder the word. It also counts sentences. This sets it apart from some 
+other formulas.
```

Comment:
```suggestion
We will use the
[Automated readability index (ARI)][wiki] formula for calculating the text complexity.
The ARI was developed in 1967 and assesses the U.S. grade level required to read 
a piece of text. In some ways, it is similar to other formulas. Its main difference 
is rather than counting syllables, it counts characters. The more characters per word, 
the harder the word. It also counts the number of words per sentences. This sets it apart from some 
other formulas.
```

Should this link to other assessments?